<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogColorReplace</class>
 <widget class="QDialog" name="DialogColorReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>被替换颜色：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="CustomLabelColor" name="label_src">
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>目标颜色：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CustomLabelColor" name="label_des">
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>色差：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabelColor</class>
   <extends>QLabel</extends>
   <header>customlabelcolor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
